### PR TITLE
[16.0][ADD] edi_partner_oca

### PR DESCRIPTION
**Pull request by QuocDuong1306**

### CODE REVIEW

File: edi_partner_oca/README.rst [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/edi-framework/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us to smash it by providing a detailed and welcomed
+`feedback <https://github.com/OCA/edi-framework/issues/new?body=module:%20edi_partner_oca%0Aversion:%2016.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Do not contact contributors directly about support or help with technical issues.
+
+Credits
```
Comment (tuantrantg) [inline edi_partner_oca/README.rst:49]: @QuocDuong1306 oi, please add [CREDITS.rst](https://github.com/QuocDuong1306/edi/pull/9/files#diff-22feadc940d8006276dd166eda0a3fb73da8a35175d7a27a31e3fdc4adb33ee1) and re-generate README.rst

Comment (QuocDuong1306) [reply edi_partner_oca/README.rst:49]: Thank a Bond, I updated

File: edi_partner_oca/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Copyright 2023 ForgeFlow S.L. (http://www.forgeflow.com)
```
Comment (nilshamerlinck) [inline edi_partner_oca/__manifest__.py]: bro

Comment (nilshamerlinck) [reply edi_partner_oca/__manifest__.py]:
```
# Copyright 2024 Jacques-Etienne Baudoux (BCIM) <je@bcim.be>
# Copyright 2024 Camptocamp SA
# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
```

Comment (nilshamerlinck) [reply edi_partner_oca/__manifest__.py]: note the 2024

Comment (QuocDuong1306) [reply edi_partner_oca/__manifest__.py]: Thank @nilshamerlinck , I updated

**Pull request merged**